### PR TITLE
Us3325 ansible concourse

### PR DESCRIPTION
**Pull request by zwerheim**

### CODE REVIEW

**Comment by zwerheim** [comment]
The IP listed in the phoenix inventory file is a test VM I setup in the phoenix org. Change the IP to test it somewhere else (or try localhost)

**Comment by zwerheim** [comment]
Look at the readme in the deploy folder for instructions.

**Comment by NaviSteve** [comment]
@lathrop-navisite Most of the files on which you commented were part of the source ansible project at https://github.com/ahelal/ansible-concourse

File: deploy/test/helper_roles/fly/files/macos.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+resources:
+- name: resource-tutorial
+  type: git
+  source:
+    uri: https://github.com/starkandwayne/concourse-tutorial.git
```
Comment (zwerheim) [inline deploy/test/helper_roles/fly/files/macos.yml]: This is not needed. Deleting it.

File: deploy/ansible.cfg [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+[ssh_connection]
+
+# Improve speed requires you disable ‘requiretty’ in /etc/sudoers on all managed hosts.
+pipelining=True
+
+## SSH Args
+control_path = %(directory)s/%%h-%%r
+
+## Makes things faster but might break tunnels ... Yeah w
```
Comment (zwerheim) [inline deploy/ansible.cfg]: No idea... I noticed that too. I guess I'll just remove it?

**Pull request closed without merging**